packages/expo-media-library/android/src/main/java/expo/modules/medialibrary/MediaLibraryConstants.java

```diff
@@ -56,8 +56,8 @@
       put(SORT_BY_CREATION_TIME, MediaStore.Images.Media.DATE_TAKEN);
       put(SORT_BY_MODIFICATION_TIME, MediaStore.Images.Media.DATE_MODIFIED);
       put(SORT_BY_MEDIA_TYPE, MediaStore.Files.FileColumns.MEDIA_TYPE);
-      put(SORT_BY_WIDTH, MediaStore.Images.Media.WIDTH);
-      put(SORT_BY_HEIGHT, MediaStore.Images.Media.HEIGHT);
+      put(SORT_BY_WIDTH, MediaStore.MediaColumns.WIDTH);
```

Comment:
I don't exactly recall if this change was necessary. I think my intention was to use a media-type-agnostic identifier here, with the hope that it would help for videos

Reply:
I looked into this a bit more. These two constants are exactly the same.

I think I prefer `MediaStore.MediaColumns.WIDTH` since the name doesn't appear to be image-specific, but I can revert these changes in the interest of preserving `git blame`. Let me know!

Reply:
So, if there are the same and we also use them when we deal with `videos`, `MediaColumns` is better ;)

Thank for investigating this 🥇